Src/Public/New-AsBuiltReport.ps1

```diff
@@ -130,13 +137,12 @@ function New-AsBuiltReport {
         [String] $Username,
 
         [Parameter(
-            Position = 3,
             Mandatory = $true,
-            HelpMessage = 'Please provide the password to connect to the target system',
-            ParameterSetName = 'UsernameAndPassword'
+            HelpMessage = 'Please provide an API token to connect to the target system',
+            ParameterSetName = 'APIToken'
         )]
         [ValidateNotNullOrEmpty()]
-        [String] $Password,
```

Comment:
It appears as though the password parameter has been removed? I'm unclear as to why the password parameter was removed? Authentication options should be for 'Username & Password' or 'Credential' or 'Token', correct?

Reply:
Whooops - I must have removed it accidentally.  The lastest push as reinstated this, and moved Token below all of the other positional params